src/Behat/Testwork/Suite/Suite.php

```diff
@@ -48,4 +48,12 @@ public function hasSetting($key);
      * @return mixed
      */
     public function getSetting($key);
+
+    /**
+     * Set key value pair to the settings
+     *
+     * @param string $key
+     * @param mixed $value
+     */
+    public function setSetting($key, $value);
```

Comment:
That's a BC break. Do not add methods to the interface (the class is OK though)

Reply:
@Taluu I know that this is Bc break, but it's more convenient to relay on interface

Reply:
It may be more convenient but it's still a BC break, so you'll have to comment out this one.